### PR TITLE
signin page

### LINKED ISSUE #14: Sign in page
### sign in page 

**path** : /signin
_________
**sign in page consist of:**

-  form (input filed and button) to enable users to log into the website
___________

image

### PR DESCRIPTION
**Pull request by yasmeenattallah**
Fix #14

### CODE REVIEW

File: client/src/App/App.js [hunk 1]
```diff
@@ -34,6 +33,7 @@ import {
   Signup,
   NotFound,
 } from '../pages';
+import { UserProvider } from '../utils';
```
Comment (mFlaifel) [inline client/src/App/App.js:36]: please don't modify app.js

File: server/utilis/validation/signinValidation.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ const signinValidation = async (req, res, next) => {
   const { email, password } = req.body;
   const schema = Joi.object({
     email: Joi.string().email().required(),
-    password: Joi.string().min(6).required()
+    password: Joi.string().min(8).required()
```
Comment (mFlaifel) [inline server/utilis/validation/signinValidation.js:8]: sorry for missing this, but the minimum in signup is 6, so please discuss with @khamis-hesham if you want 6 or 8 as a minimum and modify front-end and backend validation with the agreed value

File: client/src/pages/Signin/index.js [hunk 3]
```diff
@@ -1,5 +1,137 @@
...
+
+  const handleSubmit = async (e) => {
+    try {
+      e.preventDefault();
+      setIsLoading(true);
+      const user = { email, password };
+      await Axios.post('/api/v1/signin', user);
+      clear();
+      setIsLoading(false);
+      history.push(HOME_PAGE);
```
Comment (mFlaifel) [inline client/src/pages/Signin/index.js:46]: please don't send a request when there is an error with validation.

**Review by mFlaifel — COMMENTED** [review]
image
please center the form horizontally,
if you can, remove the blue background color in the text field when autocomplete is active

**Comment by yasmeenattallah** [comment]
> image
> please center the form horizontally,
> if you can, remove the blue background color in the text field when autocomplete is active

this autofill background color is solved  In my browser

**Review by imansedky5 — DISMISSED** [review]

**Pull request merged**